packages/arb-token-bridge-ui/src/components/common/atoms/Switch.tsx

```diff
@@ -14,38 +15,42 @@ export const Switch = ({
   className,
   label,
   description,
+  name,
   checked,
   disabled,
   onChange
 }: SwitchProps) => {
   return (
     <HeadlessSwitch.Group>
-      <div className="toggle-switch flex flex-col">
+      <div className="toggle-switch flex flex-col text-white/70 hover:text-white">
```

Comment:
When a switch button has an image and is off, it looks a bit odd. It's because the image is not grayed out and gives off a feel of inconsistency.

Maybe we can use opacity instead? And then instead of making text white on hover, we could use `arb-hover`. We should keep all the hovers smooth because that's the pattern we've been following in the reskin.

<img width="259" alt="Screenshot 2024-02-13 at 14 37 24" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/47932951/75a1ee91-030e-4cf7-aa52-e6a94b05e213">